src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Services/SemanticTokensEditsDiffer.cs

```diff
@@ -1,127 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT license. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.Collections.Immutable;
-using System.Linq;
-using OmniSharp.Extensions.LanguageServer.Protocol.Models;
-using static Microsoft.AspNetCore.Razor.LanguageServer.Semantic.DefaultRazorSemanticTokensInfoService;
-
-namespace Microsoft.AspNetCore.Razor.LanguageServer.Semantic
-{
-    internal class SemanticTokensEditsDiffer : TextDiffer
```

Comment:
😭 I'll miss this code.

Reply:
Actually the differ itself was super easy to write once I realized I could use `TextDiffer` to avoid doing the hard part. I just have fond memories of thinking this would take weeks and then slamming out `SemanticTokensEditsDiffer` in < half a day lol.